lib/csv2avro/converter.rb

```diff
@@ -18,11 +21,18 @@ def initialize(reader, writer, bad_rows_writer, error_writer, options, schema: s
 
     def convert
       while not csv.eof? do
+        @log_writer.info(metrics: [Log::Metric.new('lines_processed', row_number, 'counter')]) if row_number % 1000 == 0
         begin
           row = csv.shift
         rescue CSV::MalformedCSVError
           error_msg = "L#{row_number}: Unable to parse"
-          @error_writer.puts(error_msg)
+          @log_writer.info(message: error_msg,
+                           event: Log::Event.new('parse_error',
+                                            true,
```

Comment:
Events marked as `monitored` will show up in an event stream on Datadog. There will be thousands of parse_errors and schema_violations every day, so that does not scale.

To quote the original idea:

> Datadog Events are extremely verbose as they show up in the event feed so those should be kept for high-level, business events.

They also show up as HipChat notifications, email, etc. So let's keep monitored events very few (1 per file for instance)